examples/xgboost_cv.py

```diff
@@ -0,0 +1,104 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset using XGBoost.
+
+In this example, we perform cross-validation to accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python xgboost_cv.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize xgboost_cv.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
```

Comment:
#1566 removed this instruction from the other examples. Please remove it from this file too.
```suggestion
```

Reply:
Fixed it as per https://github.com/optuna/optuna/issues/1394